htdocs/mrp/mo_production.php

```diff
@@ -967,7 +973,8 @@
 							print '<td>';
 							if ($tmpproduct->status_batch) {
 								$preselected = (GETPOSTISSET('batch-'.$line->id.'-'.$i) ? GETPOST('batch-'.$line->id.'-'.$i) : '');
-								print '<input type="text" class="width50" name="batch-'.$line->id.'-'.$i.'" value="'.$preselected.'">';
+								print '<td>'.$formproduct->selectLotStock('', 'batch-'.$line->id.'-'.$i, '', 1, 0, $line->fk_product, '', '', '', '', '', '', 1).'</td>';
```

Comment:
It seems we are loosing the $preselected parameter.

Reply:
Hi Eldy, i will try to add the new method like you sugessted but i need a few days because i'm on the road

Reply:
PR Fixed

Reply:
Can you paste a screenshot of the result ?

Reply:
![Bildschirmfoto vom 2021-10-04 19-39-44](https://user-images.githubusercontent.com/88087088/135898826-3ade0815-fbf0-4411-8432-c826694ba3ac.png)
Hi Eldy, attached the screenshot. I'm using Chrome  94.0.4606.71 on Fedora 34.
Using Chrome and Firefox on Windows also shows the dropdown with values. But it's a text field and the Browser accepts any value i entered. Best regards Stephan